Description: Corrects path to SAMtools headers in Debian.
Origin: vendor
Forwarded: no
 Not checked yet it if it still relevant for next upstream release.

diff --git a/seqan/store.h b/seqan/store.h
index b558e94..de54ace 100644
--- a/seqan/store.h
+++ b/seqan/store.h
@@ -47,8 +47,8 @@
 #endif  // #ifndef SEQAN_HAS_SAMTOOLS 
 
 #if SEQAN_HAS_SAMTOOLS
-#include <sam.h>
-#include <bam.h>
+#include <samtools/sam.h>
+#include <samtools/bam.h>
 #endif  // #if SEQAN_HAS_SAMTOOLS
 
 //////////////////////////////////////////////////////////////////////////////
